templates/taxi/driver_list.html

```diff
@@ -7,7 +7,10 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
Ensure that the `driver-detail` URL correctly resolves using `driver.id` as the primary key. This is important for linking to the driver's profile page as specified in the task requirements.